dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -2050,7 +2072,7 @@ $modal-no-icon-gap: 22px;
   line-height: 20px;
 }
 
-#again-button {
```

Comment:
It looks like this CSS rule may be used elsewhere? https://github.com/code-dot-org/code-dot-org/search?q=again-button